### PR TITLE
Added heart reaction feature

### PR DESCRIPTION
**Pull request by KingRain**
Closes https://github.com/jitsi/jitsi-meet/issues/15211

Added the heart reaction icon to the Reaction pane and also the necessary audio files for the reaction.

### CODE REVIEW

File: react/features/reactions/constants.ts [hunk 1]
```diff
@@ -4,7 +4,8 @@ import {
     LAUGH_SOUND_FILES,
     LIKE_SOUND_FILES,
     SILENCE_SOUND_FILES,
-    SURPRISE_SOUND_FILES
+    SURPRISE_SOUND_FILES,
+    HEART_SOUND_FILES
```
Comment (aaronkvanmeerten) [inline react/features/reactions/constants.ts:8]: please add this into the middle to preserve alphabetic sorting.

**Comment by damencho** [comment]
What is the origin of the sound files?

**Comment by KingRain** [comment]
> What is the origin of the sound files?

They are from https://uppbeat.io/ made by Epic Stock Media

**Review by saghul — DISMISSED** [review]
LGTM! (untested though)

**Comment by damencho** [comment]
Drop the last commit, it is not relevant to this feature.

**Review by aaronkvanmeerten — APPROVED** [review]
LGTM (also didn't test)

**Comment by damencho** [comment]
jenkins test this please.

**Review by damencho — CHANGES_REQUESTED** [review]
This is not looking good, it used to be one line ... 
image
Also, it is missing the tooltip test: 
image

Other than that LGTM.

**Comment by damencho** [comment]
jenkins test this please.

**Review by damencho — APPROVED** [review]

**Comment by damencho** [comment]
Cheers! Thank you for your contribution.

**Comment by bgrozev** [comment]
Thanks for the contribution! Please use ~/resources/encode-sound.sh to encode the files next time (to keep the size smaller and formats consistent)

**Pull request merged**